### ISSUE TITLE
InfiniteScroll is broken

### ISSUE THREAD

**Issue by bond95** [oVirt/ovirt-web-ui#708]
InfiniteScroll start to fetch data in begining and continue even if no data is returned.

**Comment by gregsheremeta** [comment]
Was this introduced in main page updates, or has it always been a problem?

**Comment by bond95** [comment]
Not exactly, in main page branch everything worked fine. But after merge with the latest master it broke.

**Comment by bond95** [comment]
@gregsheremeta Also it can freeze whole application. Thus I think it is not `medium` task.
